nixos/modules/services/networking/gobgpd.nix

```diff
@@ -0,0 +1,73 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.gobgpd;
+  format = pkgs.formats.toml { };
+in {
+  options.services.gobgpd = {
+    enable = mkEnableOption "Whether to enable GoBGP Routing Daemon.";
+
+    package = mkOption {
```

Comment:
It appears we only have 1 version of this package and there are no variants or compile options exposed for it. Does this option provide any value?

Reply:
No. This option is not necessary. Thank you.